parm/soca/obsprep/obsprep_config.yaml

```diff
@@ -149,3 +149,21 @@ observations:
     binning:
       stride: 15
       min number of obs: 10
+- obs space:
+    name: insitu_profile_bathy
+    provider: GTS
+    dmpdir subdir: atmos
+    output file: '*.bathy_profile.ioda.nc'
```

Comment:
Maybe we don't need to spend too much time on this, since most of these yaml keys are for the netcdf to ioda converters. I have no idea what's needed for the bufr to ioda conversion.
Any reason why you name the output file starting with a wildcard?

Reply:
No reason but followed what was put in for metop output file.
The output file for bufr2ioda is prescribed as: `iodafile = f"{cycle_type}.t{hh}z.{data_type}_profiles.{data_format}.nc"`
The environment variables are from the json file (e.g., bathythermal):
`{
  "data_format"      : "bathy",
  "subsets"          : "BATHY",
  "source"           : "NCEP data tank",
  "data_type"        : "bathy",
  "cycle_type"       : "{{ RUN }}",
  "cycle_datetime"   : "{{ current_cycle | to_YMDH }}",
  "dump_directory"   : "{{ DMPDIR }}",
  "ioda_directory"   : "{{ COM_OBS }}",
  "data_description" : "6-hrly in situ Bathythermal profiles",
  "data_provider"    : "U.S. NOAA"
}`

Reply:
The `output file` name is a placeholder, so I named an output file for bufr based on what was done.
We can have a consensus on what the filename should be or if you already have one, then the `output file` name can be updated.

Reply:
Then just remove the `output file` key if it's not used @ShastriPaturi . Or just don't put a `*` in front of what is supposed to be a filename.

Reply:
@guillaumevernieres, the `*` is removed from the `output file` yaml key.